docs/index.rst

```diff
@@ -62,3 +59,22 @@ General Documentation
    getting_started
    tutorials/index
    api
+
+
+.. _astroplan_authors:
+
+Authors
+=======
+
+Maintainers
```

Comment:
I'm not sure that it's useful to separate maintainers from contributors?
What would be the criterion for this for the next few years ... when does someone start to be a maintainer or stop being a maintainer?